pkg/group_snapshotter/group_snapshotter.go

```diff
@@ -93,6 +93,19 @@ func (gs *groupSnapshot) DeleteGroupSnapshot(ctx context.Context, groupSnapshotI
 }
 
 func (gs *groupSnapshot) GetGroupSnapshotStatus(ctx context.Context, groupSnapshotID string, snapshotterListCredentials map[string]string) (bool, time.Time, error) {
-	// TODO: Implement GetGroupSnapshotStatus
-	return true, time.Now(), nil
+	klog.V(5).Infof("CSI GetGroupSnapshotStatus: %s", groupSnapshotID)
+	client := csi.NewGroupControllerClient(gs.conn)
+
+	req := csi.GetVolumeGroupSnapshotRequest{
+		GroupSnapshotId: groupSnapshotID,
+		Secrets:         snapshotterListCredentials,
```

Comment:
`repeated string snapshot_ids` is also a required field.

Reply:
Indeed, and I'm trying to find out how/where a user will provide this list.

`VolumeGroupSnapshotContentSource` only contains the `VolumeGroupSnapshotHandle`, not a list that can be used for `snapshot_ids`.

https://github.com/kubernetes-csi/external-snapshotter/blob/a9b026a8df3f9b90f2319582421067c49693f12f/client/apis/volumegroupsnapshot/v1alpha1/types.go#L347-L363

Reply:
Take a look at how DeleteGroupSnapshot is implemented.  It requires snapshot_ids as well.  You'll need to add snapshot_ids as input parameter of DeleteGroupSnapshot function.

groupsnapshot_helper.go -> csi_handler.go -> group_snapshotter.go

Reply:
Right. But DeleteGroupSnapshot() fetches the `snapshot_ids` through VolumeGroupSnapshotContentStatus.VolumeSnapshotContentRefList. Using that for GetGroupSnapshotStatus() would require users to specify VolumeGroupSnapshotContent.status in their hand-crafted yaml files. I do not think that is expected (is it even possible?).

Is there an example `VolumeGroupSnapshotContent` that is intended for importing a pre-provisioned volumegroup-snapshot? If I understand better what users are expected to provide, I should be able to fit that into the GetGroupSnapshotStatus() call somehow.

I have been using something like this for testing, without knowing where to put the `snapshot_ids`:
```yaml
---
apiVersion: groupsnapshot.storage.k8s.io/v1alpha1
kind: VolumeGroupSnapshotContent
metadata:
  name: manual-vgs-content
spec:
  deletionPolicy: Delete
  driver: hostpath.csi.k8s.io
  source:
    volumeGroupSnapshotHandle: b4e94bdc-8aca-11ee-bd04-0a580a8002f5
  volumeGroupSnapshotClassName: csi-hostpath-vgsc
  volumeGroupSnapshotRef:
    apiVersion: groupsnapshot.storage.k8s.io/v1alpha1
    kind: VolumeGroupSnapshot
    name: manual-vgs
    namespace: default
```

Reply:
FYI, we are making an API change to include `VolumeSnapshotHandles []string` in the VolumeGroupSnapshotContent: 
https://github.com/kubernetes-csi/external-snapshotter/pull/971

Reply:
Thanks for the reconsidering this detail in the API!